libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -35,17 +36,336 @@ const selector = 'eo-landing-page-shell';
+          Imagine if we all knew, where our energy came from, at all times, and
+          were able to choose the green energy by the hour. That is the vision,
+          we are working for with the platform Energy Origin. <br /><br />
+          Simultaneity between production of sustainable energy and consumption
+          of energy will be a real factor in the green transition and therefore
+          has great potential for future green solutions.
+        </p>
+      </div>
+      <div class="${selector}__column--large">
+        <img
+          class="${selector}__img"
+          src="/assets/graph-of-energy-with-dashboard.png"
+          alt="EnergyOrigin graph of energy"
+        />
+      </div>
+    </div>
+
+    <div class="${selector}__content ${selector}__display-flex">
+      <div class="${selector}__column--large">
+        <img
+          class="${selector}__img"
+          src="/assets/landing-page-office-people.jpg"
+          alt="EnergyOrigin"
+        />
+      </div>
+      <div class="${selector}__column--small watt-space-inset-l">
+        <h2 class="${selector}__h2">Who is it for</h2>
+        <p class="${selector}__p">
+          This first version of Energy Origin is for companies in Denmark and
+          can be used for e.g.:
+        </p>
+        <ul>
+          <li>Compiling an emissions overview for your annual ECG report</li>
+          <li>
+            Gaining an overview of the renewables share of your energy
+            consumption
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="${selector}__content ${selector}__display-flex space-xl">
+      <div
+        class="${selector}__column--small watt-space-inset-l eo-padding-left-none"
+      >
+        <h2 class="${selector}__h2">Who are we</h2>
+        <p class="${selector}__p">
+          Energinet is an independent public enterprise owned by the Danish
+          Ministry of Climate and Energy. We own, operate and develop the
+          transmission systems for electricity and natural gas in Denmark.
+        </p>
+      </div>
+      <div class="${selector}__column--large">
+        <img
+          class="${selector}__img"
+          src="/assets/landing-page-energy-origin-energi-huset.jpg"
+          alt="EnergyOrigin - Energihuset"
+        />
+      </div>
+    </div>
+
+    <div class="${selector}__wrapper ${selector}__wrapper--highlighted">
+      <div class="${selector}__content watt-space-inset-xl eo-text-center">
+        <h2 class="${selector}__h2">Login with your business NemId</h2>
+        <a
+          aria-labelledby="Start"
+          class="${selector}__login-link"
+          mat-button
+          mat-flat-button
+          *rxLet="loginUrl$ as loginUrl"
+          [href]="loginUrl"
+          >Start</a
+        >
+      </div>
+    </div>
+
+    <img
+      src="/assets/landing-page-mesh-bottom.png"
+      class="${selector}__footer-mesh"
+    />
+
     <eo-footer></eo-footer>
   `,
```

Comment:
issue: This component's template and styles are now way too big. It's time to divide this component into smaller sub-components. Shell components are not meant to do much more than have routes and/or child components in their template.

Reply:
Updated